### PR TITLE
Remove remnants of express-session

### PR DESCRIPTION
**Pull request by nwalters512**
This will allow `cookie` to be updated, resolving a security warning.

### CODE REVIEW

File: packages/flash/package.json [hunk 1]
```diff
@@ -19,7 +19,6 @@
   },
   "devDependencies": {
     "@prairielearn/tsconfig": "workspace:^",
-    "@types/express-session": "^1.18.0",
```
Comment (jonatanschroeder) [inline packages/flash/package.json:22]: Does this need a changelog? I'm ok with a no, just making sure this is considered.

File: packages/session/package.json [hunk 2]
```diff
@@ -15,7 +15,7 @@
   },
   "dependencies": {
     "@types/express": "^4.17.21",
-    "cookie": "^0.6.0",
+    "cookie": "^0.7.1",
```
Comment (jonatanschroeder) [inline packages/session/package.json:18]: Does this update require a changelog?

**Review by jonatanschroeder — APPROVED** [review]
It seems the README for the flash package still references express-session. While the comments there are generic, I wonder if they should point to @prairielearn/session instead.

File: apps/prairielearn/src/pages/authLogout/authLogout.ts [hunk 3]
```diff
@@ -22,9 +22,6 @@ router.get(
       setCookie(res, ['pl_disable_auto_authn', 'pl2_disable_auto_authn'], '1');
     }
 
-    // Hold-over from the old express-session implementation
-    clearCookie(res, 'connect.sid');
-
```
Comment (nwalters512) [inline apps/prairielearn/src/pages/authLogout/authLogout.ts:27]: I just made this change. Any stored cookies would have expired long ago, and we're certainly not reading this cookie anymore.

File: packages/flash/package.json (hunk 1 above)
Comment (nwalters512) [reply packages/flash/package.json:22]: Added changelog in 7d8bcbc.

File: packages/session/package.json (hunk 2 above)
Comment (nwalters512) [reply packages/session/package.json:18]: Added changelog in 7d8bcbc.

File: packages/flash/README.md [hunk 4]
```diff
@@ -4,7 +4,7 @@ Adds support for flash messages to Express applications.
 
 ## Usage
 
-`req.session` must exist; [`express-session`](https://www.npmjs.com/package/express-session) is the most common way to use sessions in Express.
+`req.session` must exist. In PrairieLearn, this is provided by the [`@prairielearn/session`](../session/README.md) package. In other applications, [`express-session`](https://www.npmjs.com/package/express-session) is a common way to implement sessions in Express applications.
```
Comment (jonatanschroeder) [inline packages/flash/README.md:7]: Nit: would the link to `../session/README.md` work in the npmjs description that reads this file?

**Review by jonatanschroeder — APPROVED** [review]

File: packages/flash/README.md (hunk 4 above)
Comment (nwalters512) [reply packages/flash/README.md:7]: According to https://github.com/npm/feedback/discussions/210, this should just work. Let's see if that's the case!

**Pull request merged**